src/content/components/VirtualList.js

```diff
@@ -245,6 +245,7 @@ VirtualList.propTypes = {
   focusable: PropTypes.bool.isRequired,
   specialItems: PropTypes.array.isRequired,
   onKeyDown: PropTypes.func.isRequired,
+  onCopy: PropTypes.func.isRequired,
```

Comment:
We could make these two props optional without additional work, I think. AFAIK in React having attribute={undefined} is like not specifying the attribute at all.